ratis-logservice/src/main/java/org/apache/ratis/logservice/server/LogServer.java

```diff
@@ -61,13 +64,31 @@ public static Builder newBuilder() {
         return new Builder();
     }
 
+    @Override
+    void setRaftProperties(RaftProperties properties) {
+      super.setRaftProperties(properties);
+
+      // Increase the client timeout
+      RaftClientConfigKeys.Rpc.setRequestTimeout(properties, TimeDuration.valueOf(100, TimeUnit.SECONDS));
+
+      // Increase the segment size to avoid rolling so quickly
+      SizeInBytes segmentSize = SizeInBytes.valueOf("32MB");
```

Comment:
why hardcoding? can't we do it through some config? or it is just default and will be overridden in later step?

Reply:
This one was weird. I was playing with making it configurable, but changes to it had some _really_ harmful implications (things were just hanging when I increased this, for example).

I left it here as something for us to come back to later, but not something I wanted to have people tweak. Maybe it's OK to still let this be configurable? (with a big-fat-warning :P)